spec/models/petition_spec.rb

```diff
@@ -2632,8 +2636,8 @@
           expect(petition.state).to eq(Petition::CLOSED_STATE)
         end
 
-        it "sets the closed date to now" do
-          expect(petition.closed_at).to be_within(1.second).of(now)
+        it "does note set the closed date" do
+          expect(petition.closed_at).to eq(nil)
```

Comment:
I think rubocop-rspec would say to use `be_nil` here but it's not installed so I'll leave it up to you 🙈